src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgery.cs

```diff
@@ -97,5 +97,14 @@ public void Validate([NotNull] HttpContext context, AntiForgeryTokenSet antiForg
         {
             Validate(context, antiForgeryTokenSet.CookieToken, antiForgeryTokenSet.FormToken);
         }
+
+        /// <summary>
+        /// Generates an anti-forgery cookie token for this request. 
```

Comment:
Generates and sets an .... if one is not available yet (or something along these lines)